stdlib/solana_sdk.h

```diff
@@ -272,6 +274,10 @@ static uint64_t sol_deserialize(
         input = (uint8_t *)(((uint64_t)input + 8 - 1) & ~(8 - 1)); // padding
         input += sizeof(uint64_t);
       }
+      else
+      {
+        input += 7; // padding
```

Comment:
Could mention why there is the padding (and/or why it is 7bytes) in the comment too